test/unit/test-fixed-layer.js

```diff
@@ -1830,5 +1831,26 @@ describes.sandboxed(
       const executed = fixedLayer.setup();
       expect(executed).to.be.true;
     });
+
+    it('should call Animation.animate in animateFixedElements', () => {
+      const animateSpy = window.sandbox.spy(Animation, 'animate');
+
+      ampdoc = new AmpDocSingle(win);
+      installPlatformService(win);
+      installTimerService(win);
+      installViewerServiceForDoc(ampdoc);
+      viewer = Services.viewerForDoc(ampdoc);
+      viewer.isEmbedded = () => true;
+
+      const fixedLayer = new FixedLayer(
```

Comment:
Did you validate this fails when the import is removed?

Reply:
Yes, this will not work when the import is removed.
I also removed `Animation` in the `.eslintrc.js` and the test still failed.